models/project_board.go

```diff
@@ -100,6 +107,10 @@ func createBoardsForProjectsType(sess *xorm.Session, project *Project) error {
 
 // NewProjectBoard adds a new project board to a given project
 func NewProjectBoard(board *ProjectBoard) error {
+	if board.Color != "" && !BoardColorPattern.MatchString(board.Color) {
```

Comment:
Last tiny nit form my side:
```suggestion
	if len(board.Color) != 0 && !BoardColorPattern.MatchString(board.Color) {
```

Reply:
Done :)